test/CMakeLists.txt

```diff
@@ -152,6 +154,7 @@ if( iodaconv_bufr_ENABLED )
     testoutput/NC005066.nc
     testoutput/airep_multi.nc
     testoutput/amdar_single.nc
+    testinput/gnssro_2020-306-2358C2E6.nc
```

Comment:
I think this is the cause of the test failure. Need to change "testinput" to "testoutput" on this line.

Reply:
> I think this is the cause of the test failure. Need to change "testinput" to "testoutput" on this line.

Yup, dumb mistake and got confused by the error message regarding a missing source code file, not an error message about a missing comparison file.  Is this cause for a different issue to be created?  Thanks for catching my error.

Reply:
If you are talking about the not-so-helpful error message, it would be nice if nccmp would list the file name in the message but there doesn't seem to be an option to do that. We could issue a bug report with the nccmp developers but the message is good enough to narrow it down to the two file arguments. Not sure if it's worth pursuing this.